### PR TITLE
Fixes the AAVE yield calculations in the STETH/ETH view

### PR DESCRIPTION
**Pull request by marcinciarka**
# [Fixes the AAVE yield calculations in the STETH/ETH view](https://app.shortcut.com/oazo-apps/story/6171/optimizing-the-yield-calculation-for-steth)

Follow up to previous PR review made by @sennett - https://github.com/OasisDEX/oasis-borrow/pull/1517
  
## Changes 👷‍♀️
 - fixes the calculation done in the simulation machine
  
## How to test 🧪
 - should show proper yields in the AAVE STETH/ETH open header

### CODE REVIEW

File: features/earn/aave/constants/index.ts [hunk 1]
```diff
@@ -1 +1,5 @@
+import { RiskRatio } from '@oasisdex/oasis-actions'
+import BigNumber from 'bignumber.js'
+
 export const aaveStrategiesList = ['AAVE-STETH']
+export const aaveStETHMinimumRiskRatio = new RiskRatio(new BigNumber(1.1), RiskRatio.TYPE.MULITPLE)
```
Comment (sennett) [inline features/earn/aave/constants/index.ts:5]: Nice.

File: features/earn/aave/open/services/getSthEthSimulationMachine.ts [hunk 2]
```diff
@@ -10,32 +14,47 @@ export function getSthEthSimulationMachine(
...
+                  '90Days',
+                  '1Year',
+                  'Inception',
+                ]),
+                await getStEthYields(context.riskRatioMax!, [
+                  '7Days',
+                  '7DaysOffset',
+                  '90Days',
+                  '90DaysOffset',
+                ]),
```
Comment (sennett) [inline features/earn/aave/open/services/getSthEthSimulationMachine.ts:46]: Could parallelise here?  `const [yieldsMin, yieldsMax] = await Promise.all([getStEthYields(...), getStEthYields(...)])`

File: features/earn/aave/manage/containers/AaveManageStateMachineContext.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import { useInterpret } from '@xstate/react'
+import { env } from 'process'
 import React from 'react'
 
 import { ManageAaveStateMachine } from '../state'
 
 function setupManageAaveStateContext({ machine }: { machine: ManageAaveStateMachine }) {
-  const stateMachine = useInterpret(machine).start()
+  const stateMachine = useInterpret(machine, { devTools: env.NODE_ENV === 'development' }).start()
```
[the remark this answers is not in the document]
Comment (marcinciarka) [reply features/earn/aave/manage/containers/AaveManageStateMachineContext.tsx:8]: Allows you to use Redux Dev Tools.

**Review by sennett — DISMISSED** [review]

File: features/earn/aave/open/services/getSthEthSimulationMachine.ts (hunk 2 above)
Comment (marcinciarka) [reply features/earn/aave/open/services/getSthEthSimulationMachine.ts]: Done.

**Review by sennett — APPROVED** [review]

**Pull request merged**